eng/Signing.props

```diff
@@ -28,6 +28,14 @@
 
       <!-- apphost and comhost template files are not signed, by design. -->
       <FileSignInfo Include="apphost.exe;singlefilehost.exe;comhost.dll" CertificateName="None" />
+
+      <!-- Third-party components which should be signed.  -->
+      <FileSignInfo Include="Newtonsoft.Json.dll" CertificateName="3PartySHA2" />
+      <FileSignInfo Include="AngleSharp.dll" CertificateName="3PartySHA2" />
```

Comment:
this should be removed, it's not something we build/distribute in this repo

Reply:
Haha copy paste from aspnet core and forgot to remove, thanks.